src/searchModal.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+import $ from 'jquery';
+import _ from 'lodash';
+import __ from 'i18n';
+import layoutTpl from 'ui/searchModal/tpl/layout';
+import 'ui/searchModal/css/searchModal.css';
+import component from 'ui/component';
+import 'ui/modal';
+import 'ui/datatable';
+
+/**
+ * Creates a search modal instance
+ * @param {object} config - search modal configuration
+ * @param {string} config.query - search query to be set and triggered on component creation
+ * @param {string} config.url - search endpoint
+ * @param {object} config.events - events hub
+ */
+export default function searchModalFactory(config) {
+    const instance = component().setTemplate(layoutTpl);
+    let searchInput = null;
+    let searchButton = null;
+    let clearButton = null;
+    let running = false;
+    instance.on('render', () => renderModal());
+    instance.on('destroy', () => destroyModal());
```

Comment:
```suggestion
    instance
      .on('render', renderModal);
      .on('destroy', destroyModal);
```

Reply:
I have tried, but `prettier` keeps formating it into a single line. I have checked that the way prettier works is to compare the length of the line to the `printWidth` property of the `.prettierrc.json` and split it if the line exceeds the value. If I change it to `40`, for example, I get what you propose. So don't know how to achieve this without updating the `.prettierrc.json`. 

It seems that this behaviour was a recent [update](https://prettier.io/blog/2020/03/21/2.0.0.html) on prettier. I am ussing prettier `2.0.5`, do you know which prettier version should I use?

Reply:
The suggestion is written without formatting. 
You have a several options: commit the suggestion, format the code and commit formatting or copy/paste that part and apply formatting before commit, it's up to you :)

Reply:
ops, sorry, yesterday I was blind and didn't see the actual change (to pass directly the function instead of using an anonymous function) so did not understand the change beyond splitting it in two lines. 

I commited your proposal but on a single line, as there is no method chaining there, just a simple call.